### PR TITLE
Fix cluster and storage config wording

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 
 linkTitle: "Persistent Storage Configuration"
 weight: 2140
 ---
 ## Overview
-Persistence volume is **Must** for Kubesphere. So before installation of Kubesphere,  one **default** 
-[StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) and corresponding storage plugin should be installed on the Kubernetes cluster.
-As different users may choose different storage plugin, [KubeKey](https://github.com/kubesphere/kubekey) supports to install storage plugin by the way of 
-[add-on](https://github.com/kubesphere/kubekey/blob/v1.0.0/docs/addons.md). This passage will introduce add-on configuration for some mainly used storage plugin.
+A persistent volume is a **Must** for KubeSphere. Therefore, before the installation of KubeSphere, a **default**
```
Comment (FeynmanZhou) [inline content/en/docs/installing-on-linux/introduction/storage-configuration.md:10]: `Persistent volumes are` or `Persistent storage service`

Comment (Sherlock113) [reply content/en/docs/installing-on-linux/introduction/storage-configuration.md:10]: "Persistent volumes are" is also correct. "A persistent volume is" is correct as well, as I find the description in the official website of Kubernetes:
image

Comment (FeynmanZhou) [reply content/en/docs/installing-on-linux/introduction/storage-configuration.md:10]: Does `A persistent volume` mislead the user that the installation of KubeSphere requires only one PV?

Comment (Sherlock113) [reply content/en/docs/installing-on-linux/introduction/storage-configuration.md]: sure, changed

**Pull request merged**